esti/main_test.go

```diff
@@ -71,7 +71,7 @@ func TestMain(m *testing.M) {
 		StorageNS: "esti-system-testing",
 	}
 	if *useLocalCredentials {
-		params.AdminAccessKeyID = "AKIAIOSFDNN7EXAMPLEQ"
+		params.AdminAccessKeyID = "AKIAIOSFODNN7EXAMPLE"
```

Comment:
The access key ID was changed on purpose since it is an invalid sequence per our definitions - see GenAccessKeyID

Reply:
This breaks our lakectl doctor test